core/storage/exploration/gae_models_test.py

```diff
@@ -310,6 +310,35 @@ def test_export_data_on_nonexistent_user(self):
         }
         self.assertEqual(expected_exploration_ids, exploration_ids)
 
+    def test_reconstitute(self):
```

Comment:
This name should be specific to the expected outcome, e.g. ``test_reconstitute_excludes_deprecated_properties``

Reply:
Done